test_util/test_launch.py

```diff
@@ -0,0 +1,302 @@
+import json
+from contextlib import contextmanager
+
+import pytest
+import yaml
+from docopt import DocoptExit
+
+import ssh.tunnel
+import test_util
+from pkgpanda.util import load_json
+from test_util.helpers import Host
+from test_util.launch import get_launcher, LauncherError, main
+
+MOCK_SSH_KEY_DATA = 'ssh_key_data'
+MOCK_KEY_NAME = 'my_key_name'
+
+
+def check_cli(cmd):
+    assert main(cmd.split(' ')) == 0, 'CLI args failed! {}'.format(cmd)
```

Comment:
It's not really `CLI args failed here`. The entire process is run and compared with the `exit 0` output.

Perhaps you can say, `"Command Failed: {}".format(cmd)`